### PR TITLE
Add a unittest for the PID data.

### PR DESCRIPTION
**Pull request by nomis52**

### CODE REVIEW

**Comment by nomis52** [comment]
Hmm, I don't know why the travis results aren't showing up here.

**Comment by peternewman** [comment]
Travis wasn't switched on here:
https://travis-ci.org/profile/OpenLightingProject

Try making a tweak and pushing and it should run.

File: .travis.yml [hunk 1]
```diff
@@ -11,3 +14,4 @@ notifications:
      - "chat.freenode.net#openlighting"
     on_success: change
     on_failure: change
+
```
Comment (peternewman) [inline .travis.yml]: Remove this line please, which should also trigger a Travis build.

File: data/pid_test.py [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# pid_test.py
+# Copyright (C) 2015 Simon Newton
+
+import unittest
+import json
+import jsonspec.validators
+
+# This is from an early versin of E1.37-5.
```
Comment (peternewman) [inline data/pid_test.py]: SPaG version

**Comment by peternewman** [comment]
LGTM

**Comment by peternewman** [comment]
LGTM

**Pull request merged**